damageproto/plan.sh

```diff
@@ -0,0 +1,19 @@
+pkg_name=damageproto
+pkg_origin=core
+pkg_version=1.2.1
+pkg_description="X11 Damage C header files"
+pkg_upstream_url="https://www.x.org/"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('???')
```

Comment:
The license is in the COPYING file in the source tarball.  It looks like an older version of the MIT license:

https://fedoraproject.org/wiki/Licensing:MIT#Old_Style_with_legal_disclaimer

I'm not sure if there is a better SPDX designation for but perhaps just 'MIT' will do.